web/src/components/HeaderImage.tsx

```diff
@@ -1,21 +1,24 @@
 import React from 'react'
-import makeStyles from '@mui/styles/makeStyles'
+import { styled } from '@mui/material/styles'
+const PREFIX = 'HeaderImage'
 
-const useStyles = makeStyles({
-  logo: {
+const classes = {
+  logo: `${PREFIX}-logo`
+}
+
+const Root = styled('a')({
```

Comment:
Perhaps another dumb question, feel free to tell me to go read up on this stuff (I'm being lazy). Any reason why we would 'style' the `<a>` element like this when we don't actually apply an css to the `<a>` element and only to the underlying image?

Reply:
Oh, perhaps to get proper specificity of the css selectors?

Reply:
It's being applied -- this "styles" the `a` element as a styled component. You'll see the styled component (`Root`) is being used as the `a` element below.